### ISSUE TITLE
nerd run immediately after nerd cleanup blows up

### ISSUE THREAD

**Issue by CamSoper** [NERDStack/nerd-cli#19]
Ran a nerd run immediately after nerd clean up.  Got the following output:

```plaintext
C:\src\nerd\nerd-tutorial>nerd run
(node:10652) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Command failed: npm run build && npm run start-local-win
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! file C:\src\nerd\nerd-tutorial\package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! Unexpected end of input at 22:1
npm ERR!
npm ERR! ^
npm ERR! File: C:\src\nerd\nerd-tutorial\package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR!
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

npm ERR! Please include the following file with any support request:
npm ERR!     C:\src\nerd\nerd-tutorial\npm-debug.log
```

Further investigation yields a **package.json** that is not properly closed.

```json
{
  "name": "nerdy-movies",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "autoprefixer": "6.5.1",
    "babel-core": "6.17.0",
    "babel-eslint": "7.0.0",
    "babel-jest": "16.0.0",
    "babel-loader": "6.2.5",
    "babel-preset-react-app": "^1.0.0",
    "boiler-room-custodian": "^0.5.0",
    "case-sensitive-paths-webpack-plugin": "1.1.4",
    "chalk": "1.1.3",
    "connect-history-api-fallback": "1.3.0",
    "cross-spawn": "4.0.2",
    "css-loader": "0.25.0",
    "detect-port": "1.0.1",
    "dotenv": "2.0.0",
    "eslint": "3.8.1",
    "eslint-config-react-app": "^0.3.0",
```

Is this by design?

**Comment by tstringer** [comment]
Thanks for creating this issue! A couple of questions:

 1. What OS are you running this on?
 2. What version of the nerd cli? `nerd --version`
 3. What exactly was your workflow here? What's the repro steps? e.g. `nerd install` and then `nerd run` and then `nerd cleanup` and then `nerd run`. Or did you do `nerd install` followed by `nerd cleanup` then `nerd run`? Ultimately, just looking for repro steps here

Thanks! 👍 😄

**Comment by CamSoper** [comment]
Sorry about that, I usually write better repro steps, but it was late.  :)

1. Windows 10
2. 0.4.1
3. See below.

- nerd install
- nerd run
- nerd cleanup
- nerd run

**Comment by tstringer** [comment]
I was able to repro the problem. It looks like this is an issue with [boiler-room-custodian](https://github.com/tstringer/boiler-room-custodian) (the external package I created to handle cleanup) on Windows (unable to repro it on macOS or Linux).

Currently troubleshooting this now, will have a resolution soon!

**Comment by tstringer** [comment]
Some more information, it appears as though this only happens when you do:

1. `nerd install`
2. `nerd run`
3. `nerd cleanup`

And not when you do:

1. `nerd install`
2. `nerd cleanup`

Still digging into this, will continue troubleshooting.

**Comment by tstringer** [comment]
Thanks again, @CamSoper, for bringing this to my attention. This was just fixed in `boiler-room-custodian v0.6.2` so that fix should take effect to fix this issue. Let me know if you still run into this, but my repro was fixed with the next version of brc. Thanks! 👍 😄
